### DISCUSSION TITLE
Unable to start llama.cpp (Windows 10, Chrome) TypeError: Cannot read properties of undefined (reading 'content')

### DISCUSSION THREAD

**Discussion by pikor69** [ggml-org/llama.cpp#10347] [Q&A] [answered]
The page at http://127.0.0.1:8080 says:
TypeError: Cannot read properties of undefined (reading 'content')

What changed since yesterday when it was working? Nothing.
The last time I was able to start I tried to run a much higher content length than the model allowed and things crashed.

**Comment by slaren** [comment, marked as answer]
Maybe something in the local storage got corrupted when it crashed. Try clearing cookies & site data.

**Reply by pikor69** [reply]
Indeed! Thanks! I feel like an idiot now :)
